src/Microsoft.ML.StandardLearners/Standard/LinearPredictor.cs

```diff
@@ -343,6 +344,20 @@ public void SaveAsCode(TextWriter writer, RoleMappedSchema schema)
 
         public abstract void SaveSummary(TextWriter writer, RoleMappedSchema schema);
 
+        public IDataView GetSummaryDataView(RoleMappedSchema schema)
+        {
+            var bldr = new ArrayDataViewBuilder(Host);
+
+            ValueGetter<VBuffer<DvText>> getSlotNames =
+                (ref VBuffer<DvText> dst) =>
+                    MetadataUtils.GetSlotNames(schema, RoleMappedSchema.ColumnRole.Feature, Weight.Count, ref dst);
+
+            // Add the bias and the weight columns.
+            bldr.AddColumn("Bias", NumberType.R4, Bias);
+            bldr.AddColumn("Weights", getSlotNames, NumberType.R4, Weight);
```

Comment:
>Weight [](start = 67, length = 6)

This will try to create a scalar column with multiple rows, and will not work because the bias column has only one row.